### ISSUE TITLE
ViewPropTypes is no longer exported in react native web 0.16.3

### ISSUE THREAD

**Issue by xn3cr0nx** [kohver/react-native-touchable-scale#9]
Hello :wave:  

As specified in the title, after upgrading to react-native-web 0.16.3, web app no longer works due to `ViewPropTypes` not being exported when importing the following:

```
import { TouchableWithoutFeedback, Animated, ViewPropTypes } from 'react-native';
```

Is it possible to get rid of `ViewPropTypes` and keep compatibility?

Thanks

**Comment by skam22** [comment]
`yarn add deprecated-react-native-prop-types`

```
import { TouchableWithoutFeedback, Animated } from 'react-native';
import { ViewPropTypes } from 'deprecated-react-native-prop-types';
```

**Comment by RonnygoBOOM** [comment]
deprecated-react-native-prop-types?
Is this still the method people are using to get around this, or has touchable-scale been updated to not need this dependency?

**Comment by skam22** [comment]
Yes, it’s a very simple fix. You could also link to the PR that’s pending - but it won’t be merged until one of the maintainers accepts it.

**Comment by kohver** [comment]
Merged #11, hope this will help.
